frontend/package.json

```diff
@@ -11,33 +11,67 @@
     "url": "https://openframeworks.cc"
   },
   "engines": {
-    "node": ">=12.20.55"
+    "node": ">=21.2.0"
   },
   "scripts": {
-    "start": "electron .",
-    "build": "electron-packager . $npm_package_productName --icon=static/icon/of.icns --out=dist --ignore='dist' --ignore='readme.md' --prune --all --overwrite --electron-version=25.3.0",
-    "build:osx": "echo 'DEPRECATED: build:osx is deprecated. please use build:macos instead' && npm run build:macos",
-    "build:macos": "electron-packager . $npm_package_productName --icon=static/icon/of.icns --out=dist --ignore='dist' --ignore='readme.md' --prune --overwrite --electron-version=25.3.0 --platform=darwin --arch=universal --osxUniversal.x64ArchFiles=Contents/Resources/app/app/projectGenerator",
-    "build:macos-m1": "electron-packager . $npm_package_productName --icon=static/icon/of.icns --out=dist --ignore='dist' --ignore='readme.md' --prune --overwrite --electron-version=25.3.0 --platform=darwin --arch=arm64",
-    "build:linux32": "electron-packager . $npm_package_productName --icon=static/icon/of.icns --out=dist --ignore='dist' --ignore='readme.md' --prune --overwrite --electron-version=25.3.0 --platform=linux --arch=ia32",
-    "build:linux64": "electron-packager . $npm_package_productName --icon=static/icon/of.icns --out=dist --ignore='dist' --ignore='readme.md' --prune --overwrite --electron-version=25.3.0 --platform=linux --arch=x64",
-    "build:vs": "electron-packager . --icon=static/icon/of.ico --out=dist --ignore='dist' --ignore='readme.md' --prune --overwrite --electron-version=25.3.0 --platform=win32 --arch=ia32"
+    "start": "NODE_ENV=development electron .",
+    "start:prod": "NODE_ENV=production electron .",
+    "dist": "electron-builder",
+    "dist:mac:x64": "NODE_ENV=production electron-builder --mac --x64 ",
+    "dist:mac:arm64": "NODE_ENV=production electron-builder --mac --arm64",
+    "dist:mac:universal": "NODE_ENV=production electron-builder --mac --universal",
```

Comment:
just a heads up - this link from `build:macos` is super important:
```--arch=universal --osxUniversal.x64ArchFiles=Contents/Resources/app/app/projectGenerator",```

otherwise universal PG won't work.

Reply:
Amazing yeah was some issues with that! Updated now and all seems to be working on both platforms